CenterForOpenScience/osf.io
# Update bootstrap-tagsinput

## Purpose

Update yarn dependencies.

## Changes

- Bootstrap-tag-input had a very minor problem and wasn't maintained, so using a fork to update seems like the best option.
-  

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify adding collection metadata still works
- Verify

What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects



## Ticket
